packages/pds-ember/app/styles/pds/theme/_light-deprecated.scss

```diff
@@ -1,34 +0,0 @@
-///==================================================///
-/// DEPRECATED (will be removed in future update)    ///
-///==================================================///
-:root {
-  /* DEPRECATED (will be removed in a future update) */
-  --letter-spacing: var(--pds-letterSpacing);
-  --line-height--dense: var(--pds-lineHeight--dense);
-  --line-height: var(--pds-lineHeight);
-  --paragraph-spacing: var(--pds-paragraphSpacing);
-  --word-spacing: var(--pds-wordSpacing);
-}
-$background-color: $backgroundColor;
-$body-font-family: $fontFamily;
-$body-font-size: $fontSize;
-$code-font-family: $fontFamily--code;
-$letter-spacing: $letterSpacing;
-$line-height--dense: $lineHeight--dense;
-$line-height: $lineHeight;
-$media-lg: $media--lg;
-$media-md: $media--md;
-$media-sm: $media--sm;
-$paragraph-spacing: $paragraphSpacing;
-$size-2xl: $size--2xl;
-$size-2xs: $size--2xs;
-$size-lg:  $size--lg;
-$size-md:  $size--md;
-$size-sm:  $size--sm;
-$size-xl:  $size--xl;
-$size-xs:  $size--xs;
-$word-spacing: $wordSpacing;
-$border-color-l1: $borderColor--l1;
-$border-color: $borderColor;
-$border-radius: $borderRadius;
-$fontSize--ui: $ui-fontSize;
```

Comment:
All of these deprecated variables have been removed. HCP Sass files have already been updated in a similar fashion (see cloud-ui PR 419).